src/app/mv-wrapper/media-viewer-wrapper.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, Compiler, ViewChild, ViewContainerRef, ComponentFactoryResolver,
+  ComponentFactory, ElementRef, OnInit, Renderer2, Inject, PLATFORM_ID } from '@angular/core';
+import { isPlatformBrowser } from '@angular/common';
+import { WindowService } from '@hmcts/ccd-case-ui-toolkit/dist/shared/services/window';
+
+const MEDIA_VIEWER = 'media-viewer';
+
+@Component({
+  selector: 'ccd-media-viewer',
+  templateUrl: './media-viewer-wrapper.component.html'
+})
+export class MediaViewerWrapperComponent implements OnInit {
+
+  mediaURL = '';
+  mediaFilename = '';
+  mediaContentType = '';
+
+  public constructor(public renderer: Renderer2,
+                     private el: ElementRef,
+                     private windowService: WindowService,
+                     private readonly viewContainerRef: ViewContainerRef,
+                     private componentFactoryResolver: ComponentFactoryResolver,
+                     @Inject(PLATFORM_ID) private platformId: Object) {
+  }
+
+  async ngOnInit() {
+
+    console.log('Platform ID is ' + this.platformId);
+
+    const localStorageMedia = this.windowService.getLocalStorage(MEDIA_VIEWER);
+    if (localStorageMedia) {
+      const media = JSON.parse(localStorageMedia);
+      this.mediaURL = media.document_binary_url;
+      this.mediaFilename = media.document_filename;
+      this.mediaContentType = media.content_type;
+    }
+    this.removeSideBar();
+  }
+
+  removeSideBar() {
+    if (this.el.nativeElement) {
```

Comment:
if you want to hide the sidebar, would this work instead:
`    this.el.nativeElement.hidden = true;
`

Reply:
The issue is due to the standard gov background styling is set to gray in CCD, so when MV loads the document, the gray bg from behind is visbile from behind. 
I fixed this by setting the background of new tab opened to ffffff, which resolves the issue. 
So, unfortunately hidden = true will not hide it